content/BPIP-4.md

```diff
@@ -0,0 +1,220 @@
+---
+bpip: 4
+title:  Support price discovery 
+discussions-to: https://github.com/bosonprotocol/BPIPs/discussions/9
+status: Draft 
```

Comment:
This is the correct `status` for this document, i feel like we can this merged in and then we can start to have some final discussions to move the status along accordingly. 

great work on this ana, i feel like this will be a big enabler for the protocol going forward.